SU-HSDO/suhumsci
# fix(shs-4993): refining how link colors are applied on warbler color …

# READY FOR REVIEW

## Summary
This applies the new colors to only links on the Traditional theme / Warbler color scheme. It does not apply the colors to title, label and button links. 

## Urgency
medium

## Steps to Test
1. Visit the hs-traditional site
2. Change the color scheme to warbler
3. Ensure that the colors for links are: Link: secondary (hex #005b6c) Hover: tertiary (hex #616a5d)
4. Ensure that this change does not disturb Title links, button links or label links. 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
